src/java.base/share/classes/java/lang/Math.java

```diff
@@ -209,7 +233,9 @@ public static double asin(double a) {
      * Returns the arc cosine of a value; the returned angle is in the
      * range 0.0 through <i>pi</i>.  Special case:
      * <ul><li>If the argument is NaN or its absolute value is greater
-     * than 1, then the result is NaN.</ul>
+     * than 1, then the result is NaN.
+     * <li>If the argument is {@code 1.0}, the result is {@code +0.0}.
```

Comment:
`+0.0` is used here but in several cases below  it is written out as `positive zero`.